packages/flutter/test/widgets/slotted_render_object_widget_test.dart

```diff
@@ -219,6 +219,7 @@ class _Diagonal extends RenderObjectWidget with SlottedMultiChildRenderObjectWid
       case _DiagonalSlot.bottomRight:
         return bottomRight;
     }
+    return null;
```

Comment:
Dead code, the switch handles all possible cases.

Reply:
I mean, if `slot` were known to be a `_DiagonalSlot`, perhaps.

Reply:
Right. That is known: The mixin defines the argument type as `S`, which here is `_DiagonalSlot`. The method signature here should really be changed to:

```dart
Widget? childForSlot(_DiagonalSlot slot);
```

... and then omitting the return should be fine.

(And don't look at the blame for the "incorrect" method signature :P)

Reply:
:P done